contracts/ssnlist.scilla

```diff
@@ -300,13 +300,40 @@ end
 
 (* @dev: Adds new ssn to ssnlist. Used by admin only. *)
 (* @param ssnaddr: Address of the ssn to be added *)
+(* @param rewards: Rewards of the ssn to be added *)
+(* @param urlraw: string representing "ip:port" of the ssn serving raw api request *)
+(* @param urlapi: string representing url exposed by ssn serving public api request *)
+(* @param initiator: The original caller who called the proxy *)
+transition add_ssn (ssnaddr : ByStr20, rewards : Uint128, urlraw : String, urlapi : String, initiator : ByStr20)
```

Comment:
Should we remove `rewards` as a param and keep it to zero like other fields for the SSN?

Reply:
I think we can remove it

Reply:
Yea, will do it, thanks!